Tools/rules.bzl

```diff
@@ -19,52 +19,87 @@ The __init__.py file must be in your srcs list so that Cython can resolve
 cimports using the package layout.
 """
 
+load("@rules_cc//cc:defs.bzl", "cc_binary")
+load("@rules_python//python:defs.bzl", "py_library")
+
 def pyx_library(
         name,
-        deps=[],
-        srcs=[],
-        cython_directives=[],
```

Comment:
Why were these dropped?

Reply:
These were arguments to the cythonize tool which is no longer used. What would you expect these parameters to do in the current implementation?